packages/kununu-utils/kununu-logger/kununuLogger.spec.js

```diff
@@ -92,93 +115,117 @@ describe('Returns correct log format with json format', () => {
     const printf = customFormat;
     const value = printf.template(info);
     expect(JSON.parse(value)).toEqual({
+      level_name: 'INFO',
+      time: new Date().toISOString(),
       build: '-',
-      label: 'test',
-      level: 'info',
-      custom: true,
-      logType: 'custom_logger',
+      application: 'test',
+      channel: 'custom_logger',
+      context: {},
     });
   });
 });
 
 describe('Logs according to defined level', () => {
-  let originalEnv = process.env.MINIMUM_LOG_LEVEL;
   jest.resetModules();
   process.env.MINIMUM_LOG_LEVEL = 'debug';
-  const {logger: loggerLevelTest} = require('./index');
-
-  afterAll(() => {
-    process.env.MINIMUM_LOG_LEVEL = originalEnv;
-  });
+  const {logger: loggerLevelTest} = require('./index'); // eslint-disable-line global-require
 
   it('tries to log silly level', () => {
     loggerLevelTest.silly({
       custom: true,
-      message: 'silly - Will not be logged',
+      message: 'silly - Message will not be logged',
     });
 
     expect(spyFunc).not.toBeCalled();
-    expect(generatedLog.level).toEqual(undefined);
+    expect(generatedLog.level_name).toEqual(undefined);
   });
 
   it('tries to log debug level with log and level', () => {
     loggerLevelTest.log({
       custom: true,
-      message: 'debug - Will be logged with log and level',
+      message: 'debug - Message will be logged with log and level',
       level: 'debug',
     });
 
     expect(spyFunc).toBeCalled();
-    expect(generatedLog.level).toEqual('debug');
+    expect(generatedLog.level_name).toEqual('DEBUG');
+    expect(generatedLog.message).toEqual('debug - Message will be logged with log and level');
+    expect(generatedLog.time).toBeDefined();
```

Comment:
Don't we know the values that should be here? Maybe we should avoid `toBeDefined` and have actual values whenever possible.